### PR TITLE
Add cloud icons for remote favorite paths

### PR DESCRIPTION
**Pull request by robinkar**
This PR adds cloud icons for favorite paths for remote storages both in the navbar and in the file browser.

remote-storage-icons
The navbar items are currently somewhat ugly due to the spacing, but that is fixed in my other open PR (#3075).

### CODE REVIEW

File: apps/dashboard/app/views/files/_favorites.html.erb [hunk 1]
```diff
@@ -5,7 +5,12 @@
     <ul id="favorites" class="nav nav-pills flex-column">
       <li role="presentation" class="nav-item"><%= link_to 'Home Directory', files_path(Dir.home), class: "nav-link d bg-light" %></li>
       <% OodFilesApp.new.favorite_paths.each do |p| %>
-        <li class="nav-item"><%= link_to p.title || p.path.to_s, files_path(p.filesystem, p.path.to_s), class: "nav-link d bg-light" %>
+        <li class="nav-item">
+          <%= link_to files_path(p.filesystem, p.path.to_s), class: "nav-link d bg-light" do %>
+          <%= fa_icon "cloud", classes: '' %>
```
Comment (johrstrom) [inline apps/dashboard/app/views/files/_favorites.html.erb:10]: I've got to look this over - it looks like you've hard coded `cloud` icons for all favorite paths, so I'm either misreading it or something.

Comment (robinkar) [reply apps/dashboard/app/views/files/_favorites.html.erb:10]: Hmm, let me double check that. Seems like some code may have gone missing.

Comment (robinkar) [reply apps/dashboard/app/views/files/_favorites.html.erb]: It has indeed gone missing. Added it now.

File: apps/dashboard/app/views/files/_favorites.html.erb [hunk 2]
```diff
@@ -5,7 +5,12 @@
     <ul id="favorites" class="nav nav-pills flex-column">
       <li role="presentation" class="nav-item"><%= link_to 'Home Directory', files_path(Dir.home), class: "nav-link d bg-light" %></li>
       <% OodFilesApp.new.favorite_paths.each do |p| %>
-        <li class="nav-item"><%= link_to p.title || p.path.to_s, files_path(p.filesystem, p.path.to_s), class: "nav-link d bg-light" %>
+        <li class="nav-item">
+          <%= link_to files_path(p.filesystem, p.path.to_s), class: "nav-link d bg-light" do %>
+          <%= fa_icon "cloud", classes: '' if p.remote? %>
```
[the remark this answers is not in the document]
Comment (robinkar) [reply apps/dashboard/app/views/files/_favorites.html.erb]: Added icons for the favorite paths, and also Home Directory for consistency. Although I wonder if we could rely more directly on the OodAppLinks here.
image

**Review by johrstrom — APPROVED** [review]
Thanks!

**Pull request merged**